dev-tools/notice/dependencies.csv.tmpl

```diff
@@ -1,6 +1,6 @@
 {{- define "depInfo" -}}
 {{- range $i, $dep := . }}
-{{ $dep.Name }},{{ $dep.URL }},{{ $dep.Version | canonicalVersion }},{{ $dep.Version | revision }},{{ $dep.LicenceType }},{{ $dep.URL }}
+{{ $dep.Name }},{{ $dep.URL }},{{ $dep.Version | canonicalVersion }},{{ $dep.Version | revision }},{{ $dep.LicenceType }},
```

Comment:
Is the comma at the end required?
```suggestion
{{ $dep.Name }},{{ $dep.URL }},{{ $dep.Version | canonicalVersion }},{{ $dep.Version | revision }},{{ $dep.LicenceType }}
```

Reply:
I see the csv header `name,url,version,revision,license,sourceURL`, so I assume it's required in this particular case, if so please forget my comment

Reply:
Yes, the last column in the CSV should still be there, but empty.